src/styles/main.scss

```
@@ -1,3 +1,4 @@
+@import "./utils/variables";
```

please check the naming
```suggestion
@import "./utils/_variables";
```